src/main/ts/components/Editor.tsx

```diff
@@ -8,21 +8,30 @@ import { Bookmark, Editor as TinyMCEEditor, EditorEvent, TinyMCE } from 'tinymce
 
 type EditorOptions = Parameters<TinyMCE['init']>[0];
 
-export interface IProps {
+export type Version = `${'4' | '5' | '6' | '7'}${'' | '-dev' | '-testing' | `.${number}` | `.${number}.${number}`}`;
```

Comment:
That is pretty cool didn't know you could do this to expand all the variants. TS crazy typesystem.

Reply:
The same trick can be used for the `licenseKey`, if we want to. But only if there's a prefix in the key

Reply:
Some people might still be using `5-stable` since it was promoted in docs for a while (or `stable` if they've been around since the beginning of the cloud). That said maybe it's a good idea for them to change that.
The complete list of versions currently supported by cloud is listed here:
https://cdn.tiny.cloud/1/no-api-key/tinymce/available-versions